apps/extension/tailwind.config.js

```diff
@@ -2,6 +2,5 @@
 module.exports = {
   mode: "jit",
   darkMode: "media",
-  content: ["./**/*.tsx"],
-  plugins: [require("@tailwindcss/line-clamp")]
```

Comment:
weird, seems to still work without this - just curious, do you know why?

Reply:
line-clamp plugin is now included by default